src/App.js

```diff
@@ -14,11 +15,48 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    goodsList: [...goodsFromServer],
+    show: false,
+    showBtn: true,
```

Comment:
don't save goods in the state
use only one boolean for rendering button and goods